tools/set-import/index.js

```diff
@@ -52,7 +53,7 @@ Chat.plural = function (num, plural = 's', singular = '') {
 	return (num !== 1 ? plural : singular);
 };
 // Sigh. Yay globals!
-global.toID = require('../../.sim-dist/dex').Dex.getId;
+global.toID = require('.././.sim/dex').Dex.getId;
```

Comment:
Is this even necessary anymore? The Sim API no longer requires a global `toID`.

Reply:
I don't know; I've never worked on `set-import`. My find and replace script messed up here, though!

Reply:
Yeah, that's the first thing I noticed, but it occurs to me that you can just remove this line entirely now.

Reply:
Alright. I'll also remove it from `tools/simulate`.